cardano-cli/test/cardano-cli-golden/files/golden/help/ping.cli

```diff
@@ -1,10 +1,13 @@
-Usage: cardano-cli ping [-c|--count COUNT]
-                          ((-h|--host HOST) | (-u|--unixsock SOCKET))
-                          [-p|--port PORT]
-                          [-m|--magic MAGIC]
-                          [-j|--json]
-                          [-q|--quiet]
-                          [-Q|--query-versions]
+Usage: cardano-cli ping 
+                          ( [-c|--count COUNT]
+                            (-h|--host HOST)
+                            [-p|--port PORT]
+                            [-m|--magic MAGIC]
+                            [-j|--json]
+                            [-q|--quiet]
+                            [-Q|--query-versions]
+                          | --handshake-only (-u|--unixsock SOCKET)
```

Comment:
This incorrectly omits all the other options in this case.
This is because visibility in `optparse-appicative` can only be `Internal`, `Hidden` or `Visible`.
There is no fourth visibility to hide an option in the full description and show it in the brief description.
So we either omit these options in the brief description here, or they will be doubled in the full description.

Reply:
Hum I see, there is no good solution in the current state.

@Jimbo4350> would be it acceptable to instead:

1. Keep only one parser (e.g. `pPing`) and
2. Indicate in `-u/--unixsock` that this option requires `--handshake-only`
3. Perform the check that if `-u` is specified, then so is `--handhshake-only`, when the command starts to run? (as opposed to doing the check in the parser)